pmc.py

```diff
@@ -1,75 +1,55 @@
+from argparse import ArgumentParser, RawTextHelpFormatter
 import os
 import env_setup_tools
 
 if os.name == "posix":
     env_setup_tools.configure_posix_environment()
 
-import sys
 import pyautogui
 import keyboard
 import strings
 
+pyautogui.FAILSAFE = False  # Do not crash if pointer gets out of screen
+
 
 class PainlessMouseController:
-    def __init__(self) -> None:
-        self.SPEED = 1
-        self.VERBOSE = False
-
-        pyautogui.FAILSAFE = False  # Do not crash if pointer gets out of screen
-
-        self.KEYS = {"up": "up", "down": "down", "right": "right", "left": "left"}
-        self.__set_preferences()
-
-        if "--help" in sys.argv:
-            print(strings.HELP_TEXT)
-            sys.exit(0)
-
-    def __digit_indx(self) -> int:
```

Comment:
This method can be deleted, let the user just use --speed N
If you insist that a keyless definition of movement speed must be there, then I guess this can be implemented too